hack/build-base-images.sh

```diff
@@ -8,7 +8,15 @@ source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 # determine the correct tag prefix
 tag_prefix="${OS_IMAGE_PREFIX:-"openshift/origin"}"
 
+os::util::ensure::gopath_binary_exists imagebuilder
+
+# image builds require RPMs to have been built
+os::build::release::check_for_rpms
```

Comment:
@smarterclayton @stevekuznetsov removed the RPM dependency from the base images and verified once more that `OS_BUILD_ENV_PRESERVE="_output/local/releases:_output/local/bin" hack/env make release` works. I'll squash, rebase, and wait for https://github.com/openshift/aos-cd-jobs/pull/221 before I start Jenkins on this one. In the meantime, ptal